src/c++/perf_analyzer/docs/inference_load_modes.md

```diff
@@ -40,6 +40,39 @@ example, when using
 will to attempt to have 4 outgoing inference requests at all times during
 profiling.
 
+## Periodic Concurrency Mode
+
+In periodic concurrency mode, Perf Analyzer will periodically launch a new set
+of inference requests until the total number of inference requests that has been
+launched since the beginning reaches N requests.
+
+For example, when using `--periodic-concurrency-range 10:100:20`, Perf Analyzer
+will start with 10 concurrent requests and for every step, it will launch 20 new
+inference requests until the total number of requests launched since the
+beginning reaches 100. Additionally, the user can also specify *when* to launch
+the new requests by specifying `--request-period M`. This will set Perf Analyzer
+to launch a new set of requests whenever the *last* request (among the set of new
+requests) receives M number of responses back from the server.
+
+The user can also specify custom parameters to the model using
+`--request-parameter <name:value:type>` option.
+For instance, passing `--request-parameter max_tokens:256:uint` will add
```

Comment:
Did we get consensus on the list of acceptable data type strings? `uint` seems vague. Shouldn't it include the number of bits. To get parity with whatever rapidjson or protobuf types there are?

Reply:
Looking at the [definition of `InferParameter`](https://github.com/triton-inference-server/common/blob/468eb216b01b375b83590b1e206d1c8cf4160b38/protobuf/grpc_service.proto#L439), seems like we are currently support: `bool`, `int64`, `string`, `double`, and `uint64`. 

I can add `double` since that one seems to be missing. I don't know if specifying number of bits is necessary since we are only supporting 64 bits.

Reply:
https://kserve.github.io/website/0.10/modelserving/data_plane/v2_protocol/#parameters_1

docs seem to say only bool, string, and number. In this case, i think number is unsigned or signed ints.

Reply:
Oh nvm. In the link I attached says `double` and `uint64` are just placeholders and are just there for future use, so it's just `bool`, `int64`, and `string`.